components/symbl_ai/actions/post-audio-url/post-audio-url.mjs

```diff
@@ -0,0 +1,126 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+import languages from "../languages.mjs";
+
+export default {
+  key: "symbl_ai-post-audio-url",
+  name: "Submit Audio URL",
+  description: "Submit an Audio file by providing the URL for processing. See the doc [here](https://docs.symbl.ai/docs/async-api/overview/audio/post-audio-url).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    audioUrl: {
+      type: "string",
+      label: "Audio URL",
+      description: "The URL of the audio file to be processed.",
+      optional: false,
+    },
+    meetingName: {
+      type: "string",
+      label: "Meeting Name",
+      description: "The meeting name. The default name is set to the conversationId.",
+      optional: true,
+    },
```

Comment:
Reused `props` can be defined inside app's `propDefinitions`.
This applies to other `props` as well.

See https://github.com/PipedreamHQ/pipedream/pull/2485#discussion_r835346771